modules/serverless-broker-backing-channel-default.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+//  * serverless/admin_guide/serverless-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-broker-backing-channel-default_{context}"]
+= Configuring the default broker backing channel
+
+If you are using a channel-based broker, you can set the default backing channel type for the broker to either `InMemoryChannel` or `KafkaChannel`.
+
+.Prerequisites
+
+* You have administrator permissions on {product-title}.
+* You have installed the {ServerlessOperatorName} and Knative Eventing on your cluster.
+* If you want to use Kafka channels as the default backing channel type, you must also install the `KnativeKafka` CR on your cluster.
+
+.Procedure
+
+* Modify the `KnativeEventing` custom resource (CR) to add configuration details for the `config-br-default-channel` config map:
```

Comment:
To confirm, there's no need to apply or await anything, nor is there an obvious verification step?

Reply:
We can add an apply step, sure.
There's no obvious verification step or wait afaik.

Reply:
Added `oc apply` step and requirement to install the `oc` CLI as a prerequisite. I guess we didn't add it before because you can also update this from the web console, but we don't doc / have a request for specific steps for that, so adding this instead.